src/elementary_operators.jl

```diff
@@ -15,11 +15,28 @@ using ITensors:
   prime,
   sim,
   noprime!,
-  contract
+  contract,
+  val,
+  state,
+  ValName,
+  StateName
 using ITensorNetworks: IndsNetwork, ITensorNetwork, TreeTensorNetwork, combine_linkinds, ttn
 
 default_boundary() = "Dirichlet"
 
+# reuse Qudit definitions for now
+
+function ITensors.val(::ValName{N}, ::SiteType"Digit") where {N}
+  return parse(Int, String(N)) + 1
+end
+
+function ITensors.state(::StateName{N}, ::SiteType"Digit", s::Index) where {N}
```

Comment:
See comment below on `delta` functions. We should move these helper functions to `utils.jl` for now